packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js

```diff
@@ -183,6 +194,26 @@ describe('CheckoutModule', () => {
             });
         });
 
+        describe(`${UPDATE_IS_FULFILLABLE} ::`, () => {
```

Comment:
Could all the tests from line 167 and until 226 be changes with one like this?: 
```
describe ('Mutation', () => {
    it.each([
        ['UPDATE_FULFILMENT_ADDRESS', address]
        ['UPDATE_IS_FULFILLABLE', isFulfillable],
        ['UPDATE_ISSUES', issues],
        etc
    ])(' "%s" should update state with received value'), (
    mutationName, value) => {
    }
})
```

Reply:
@dandel10n It is possible, but you would need to add an extra (string) variable for referencing the state which I think is a little untidy.

```javascript
it.each([
    [UPDATE_FULFILMENT_ADDRESS, 'address', address],
    [UPDATE_FULFILMENT_TIME, 'time', time],
    [UPDATE_IS_FULFILLABLE, 'isFulfillable', isFulfillable],
    [UPDATE_ISSUES, 'issues', issues],
    [UPDATE_USER_NOTE, 'userNote', userNote]
])('%s :: should update state with received value', (mutationName, propertyName, propertyValue) => {
    // Arrange & Act
    mutations[mutationName](state, propertyValue);

    // Assert
    expect(state[propertyName]).toEqual(propertyValue);
});
```

Reply:
I don't think that it looks untidy. In my opinion, this way looks much more maintainable and readable for me than adding a new test for every mutation. We can check all the mutations we have with one test and will need to add only one new line later for new mutation.